lib/cli/RenderAction/utils/browser/index.tsx

```diff
@@ -1,16 +1,14 @@
-import type { RenderToReadableStreamOptions } from "react-dom/server";
-import {
-  DEFAULT_WAAVY_HYDRATION_SELECTOR,
-  DEFAULT_WAAVY_PROPS_CACHE_KEY,
-} from "@/constants";
+import React from "react";
+import { renderToString } from "react-dom/server";
+
+import { DEFAULT_WAAVY_HYDRATION_SELECTOR, DEFAULT_WAAVY_PROPS_CACHE_KEY } from "@/constants";
 import type { RenderActionOptions } from "@/types";
 import { asOptionalNumber, getVersion } from "@/utils";
-import PeerDependencyManager from "@/utils/models/PeerDependencyManager";
 import { getErrorPageMarkup } from "../errors";
 
 type CreateRenderOptionsConfig = {
   bootstrap?: string[];
-  ErrorComponent?: React.ComponentType<{ error: unknown }> | null;
+  ErrorComponent?: any;
```

Comment:
Change this back to React.ComponentType<{ error: unknown; errorInfo: unknown }>;